ppr-ui/src/components/tables/RegistrationTable.vue

```diff
@@ -513,13 +513,18 @@ export default defineComponent({
       const draftItem = item as DraftResultIF
       const firstBaseReg = localState.registrationHistory[0].baseRegistrationNumber
       const firstDocId = localState.registrationHistory[0].documentId
-      if (item.baseRegistrationNumber && item.baseRegistrationNumber === firstBaseReg) {
-        return true
-      }
-      if (!item.baseRegistrationNumber && draftItem.documentId === firstDocId) {
-        return true
-      }
-      return false
+      return item.baseRegistrationNumber
+        ? item.baseRegistrationNumber === firstBaseReg
+        : draftItem.documentId === firstDocId
+    }
+
+    const isFirstItemMhr = (item: MhRegistrationSummaryIF | MhrDraftIF): boolean => {
```

Comment:
When calling `isFirstItemMhr(item as MhRegistrationSummaryIF)` you pass item only as a `MhRegistrationSummaryIF` type, so `MhrDraftIF` is not needed here.

Reply:
good point, I updated the function call to reference `item as MhRegistrationSummaryIF | MhrDraftIf` to be more type accurate. I also removed draftItem assignment, as its not necessary.